### DISCUSSION TITLE
Too much censored

### DISCUSSION THREAD

**Discussion by jarraarudhrakumar** [community/community#138150] [Other Feature Feedback, Questions, & Ideas]
Ai
Is censored the future
Without uncensored modal there is no use. Ai will be in loop. No changes, u see only minor changes in ai, which is not useful for users, developers, researchers. As in democracy(freedom of speech)countries there is no use of censored model too much limits. Worst experience on ai modal. Ai will ahead of humans: No

**Comment by PrasanthVijayy** [comment]
@jarraarudhrakumar 

AI censorship aims to ensure ethical and responsible use, but excessive limitations can hinder innovation and research. While freedom of speech is important, striking a balance between safety and openness is key. Over-censorship can indeed restrict the potential of AI, but completely uncensored models could pose risks. Finding the right balance is crucial for AI's future development and utility.

**Reply by jarraarudhrakumar** [reply]
Humans are more threatened than ai as what I see online. And humans are creating ai for our own existence. I mean (next to humans revolution there is ai revolution or hybrid (human+ai)). This cannot be stopped.

**Reply by jarraarudhrakumar** [reply]
Uncensored is risky, but it gives  innovations, creative, development for developer, researcher, users, for optimum technology than censored. U can ask ai or self think.
